src/bundle/Resources/public/js/scripts/embedded.item.actions.js

```diff
@@ -0,0 +1,318 @@
+(function (global, doc, ibexa, Routing, Translator, Popper) {
+    const MIN_ITEMS_NUMBER_TO_SHOW_SEARCH = 10;
+    const MENU_PROPS = {
+        placement: 'bottom-start',
+        fallbackPlacements: ['bottom-start', 'top-end', 'top-start'],
```

Comment:
Does default placement ('bottom-start' in this case) has to be in fallbackPlacements?